pyhive/tests/test_sqlalchemy_hive.py

```diff
@@ -112,32 +134,32 @@ def test_reflect_select(self, engine, connection):
     @with_engine_connection
     def test_type_map(self, engine, connection):
         """sqlalchemy should use the dbapi_type_map to infer types from raw queries"""
-        row = connection.execute('SELECT * FROM one_row_complex').fetchone()
+        row = connection.execute(text('SELECT * FROM one_row_complex')).fetchone()
         self.assertListEqual(list(row), _ONE_ROW_COMPLEX_CONTENTS)
 
     @with_engine_connection
     def test_reserved_words(self, engine, connection):
         """Hive uses backticks"""
         # Use keywords for the table/column name
-        fake_table = Table('select', MetaData(bind=engine), Column('map', sqlalchemy.types.String))
-        query = str(fake_table.select(fake_table.c.map == 'a'))
-        self.assertIn('`select`', query)
-        self.assertIn('`map`', query)
-        self.assertNotIn('"select"', query)
-        self.assertNotIn('"map"', query)
+        fake_table = Table('select', MetaData(), Column('map', sqlalchemy.types.String))
```

Comment:
Existing test cases were failing in SQLAlchemy 1.3, 1.4. Hence adjusted those.
`
test_sqlalchemy_hive.py:146: in test_reserved_words
    self.assertIn('`select`', query)
E   AssertionError: '`select`' not found in 'SELECT "select".map \nFROM "select" \nWHERE "select".map = :map_1'

test_sqlalchemy_hive.py:147: in test_reserved_words
    self.assertIn('`map`', query)
E   AssertionError: '`map`' not found in 'SELECT "select".map \nFROM "select" \nWHERE "select".map = :map_1'

test_sqlalchemy_hive.py:148: in test_reserved_words
    self.assertNotIn('"select"', query)
E   AssertionError: '"select"' unexpectedly found in 'SELECT "select".map \nFROM "select" \nWHERE "select".map = :map_1'

test_sqlalchemy_hive.py:150: in test_reserved_words
    self.assertIn('"map"', query)
E   AssertionError: '"map"' not found in 'SELECT "select".map \nFROM "select" \nWHERE "select".map = :map_1'
`